exasol/toolbox/nox/_format.py

```diff
@@ -47,3 +46,13 @@ def fmt_check(session: Session) -> None:
     """Checks the project for correct formatting"""
     py_files = [f"{file}" for file in python_files(PROJECT_CONFIG.root)]
     _code_format(session=session, mode=Mode.Check, files=py_files)
+
+
+class XSession:
+    def run(self, *args):
+        print(f"session.run(args: {args})")
+
+
+if __name__ == "__main__":
+    session = XSession()
+    _code_format(session, Mode.Fix, [])
```

Comment:
Is this intended or did you use it for testing?

Reply:
Thanks. This was a left over, removed in latest push,